src/RuleSet/Sets/PERCS2x0Set.php

```diff
@@ -35,6 +35,11 @@ public function getRules(): array
         return [
             '@PER-CS1.0' => true,
             'concat_space' => ['spacing' => 'one'],
+            'function_declaration' => [
+                'closure_fn_spacing' => 'none',
+                'closure_function_spacing' => 'one',
+                'trailing_comma_single_line' => false,
```

Comment:
If I understand right, only `closure_fn_spacing` is non-default here.
I suggest to explicitly configure `closure_fn_spacing` only in that case, and make a TODO for fixer to change default for `closure_fn_spacing` in next major